introduction.md

```diff
@@ -0,0 +1,2 @@
+My name is Wolf and I'll be contributing to this repo as part of a Code School course.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.